### ISSUE TITLE
manyTill_ combinator

### ISSUE THREAD

**Issue by jamesdbrock** [purescript-contrib/purescript-parsing#108]
We should have the super-useful `manyTill_` combinator.

[`Combinators.manyTill_`](https://hackage.haskell.org/package/parser-combinators-1.2.1/docs/Control-Monad-Combinators.html#v:manyTill_)

```purescript
manyTill_ :: forall e m a s. Monad m => ParserT s m a -> ParserT s m e -> ParserT s m (Tuple (List a) e)
```

Also

```purescript
someTill_ :: forall e m a s. Monad m => ParserT s m a -> ParserT s m e -> ParserT s m (Tuple (List a) e)
```

**Comment by jamesdbrock** [comment]
I've written a draft implementation of `manyTill_` and `many1Till_` and I'll submit a PR if you like.

https://github.com/jamesdbrock/purescript-parsing-string-replace/blob/be8c27680bccd1a8f54c5c98d72747a1cc0e4a84/src/Text.Parsing.Parser.String.Replace.purs#L45

**Comment by jamesdbrock** [comment]
Resolved by #143
